phpunit-integration-legacy-solr.xml

```diff
@@ -53,6 +53,7 @@
             <file>eZ/Publish/API/Repository/Tests/ObjectStateServiceAuthorizationTest.php</file>
             <file>eZ/Publish/API/Repository/Tests/SearchServiceTest.php</file>
             <file>eZ/Publish/API/Repository/Tests/SearchServiceFulltextTest.php</file>
+            <file>eZ/Publish/API/Repository/Tests/SearchServiceFulltextEmbedTest.php</file>
```

Comment:
Please include `<directory>` instead based on the new namespace I've mentioned in the beginning.

Reply:
Added and moved single entires